src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int colorIndex = new Random().nextInt(Color.values().length - 1);

```

Comment:
```suggestion
        int colorIndex = new Random().nextInt(Color.values().length);
```
this upper bound is exclusive